docker/README.md

```diff
@@ -2,55 +2,140 @@
 
 ## Prerequisites & Troubleshooting
 
-Before attempting to build openlibrary using the Docker instructions below, please follow this checklist. If you encounter an error, this section may serve as a troubleshooting guide:
+The openlibrary repository must be cloned with `ssh` and *not* `https`. See the [Git Cheat Sheet](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet) for more, including how to fix this if you use `git clone https://...`
```

Comment:
```suggestion
The openlibrary repository _must_ be cloned with `ssh` and *not* `https` so that git submodules can be fetched correctly. See the [Git Cheat Sheet](https://github.com/internetarchive/openlibrary/wiki/Git-Cheat-Sheet) for more, including how to fix this if you use `git clone https://...`
```